data/governments.txt

```diff
@@ -193,6 +193,12 @@ government "Deep Security"
 government "Derelict"
 	"fine" 0
 
+government "Derelict (Boardable)"
+	"display name" "Derelict"
+	"player reputation" -1
```

Comment:
We should make the derelict government hostile to everyone by default. Since the fleets are `marked`, it will still only be the player who can target them, but this means that stray projectiles from any ship can damage them.
```suggestion
	"player reputation" -1
	"default attitude" -.01
```